bruno.docs/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Consider adding a description or comments to explain the purpose of this file and its content. [medium]